portfolio/src/App.tsx

```diff
@@ -1,21 +0,0 @@
-import React, {Component} from 'react';
-import logo from './logo.svg';
-import './App.css';
-import Cover from './components/Cover';
-import Info from './components/Info';
-import Container from 'react-bootstrap/Container';
-import Projects from './components/Projects';
```

Comment:
One suggestion I've seen from my work is to organize the imports such that they're alphabetical and also the dependencies are first